rfd/0018/README.md

```diff
@@ -76,7 +77,11 @@ What we propose here is to make a reserved namespace for labels that have
 special meanings for sdc-docker. These would look like:
 
 ```
-com.joyent.<key>=<value>
+com.joyent.<key>=<value>     # Joyent/Triton-specific use cases
+```
+or
+```
+triton.<key>=<value>         # Triton-specific use cases
```

Comment:
This is just my attempt to accommodate both in this one RFD since we ended up using com.joyent.xxx for packages and triton.xxx for network. This also aligns with the proposed change to the sdc-docker divergence doc (https://cr.joyent.us/#/c/1803/4/docs/api/divergence.md).